airflow/models/taskinstance.py

```diff
@@ -678,6 +670,8 @@ def _execute_task(task_instance: TaskInstance, context: Context, task_orig: Oper
             )
 
     def _execute_callable(context: Context, **execute_callable_kwargs):
+        from airflow.utils.context import context_get_outlet_events
```

Comment:
Is this because context imports into the sdk? Maybe we should avoid doing it there instead.

Reply:
Yes, that.

But also: I need to remove all concept of execution from taskinstance.py as well!

So much to do!